cli/src/main/kotlin/com/amazon/ionschema/cli/commands/ValidateCommand.kt

```diff
@@ -0,0 +1,124 @@
+package com.amazon.ionschema.cli.commands
+
+import com.amazon.ion.IonStruct
+import com.amazon.ion.IonText
+import com.amazon.ion.IonType
+import com.amazon.ion.system.IonSystemBuilder
+import com.amazon.ionschema.AuthorityFilesystem
+import com.amazon.ionschema.IonSchemaSystemBuilder
+import com.amazon.ionschema.ResourceAuthority
+import com.amazon.ionschema.Schema
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.arguments.convert
+import com.github.ajalt.clikt.parameters.groups.OptionGroup
+import com.github.ajalt.clikt.parameters.groups.defaultByName
+import com.github.ajalt.clikt.parameters.groups.groupChoice
+import com.github.ajalt.clikt.parameters.groups.mutuallyExclusiveOptions
+import com.github.ajalt.clikt.parameters.options.check
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.multiple
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+class ValidateCommand : CliktCommand(
+    help = "Validate Ion data against a schema.",
+    epilog = """
+        ```
+        Example usage:
+            Validate against a new type:
+                ion-schema-cli validate --type '{ codepoint_length: range::[min, 10] }' 'hello'
+            Validate against a type from a schema:
+                ion-schema-cli validate -a file-system --base-dir ~/my_schemas/ --schema 'Customers.isl' --type 'online_customer' '{ foo: bar }'    
```

Comment:
You should extract a constant here, or somehow access this value from `name` in `IonSchemaCli`. I couldn't figure out how to do the latter, so in the copy of this that I borrowed from you I referenced a single constant in both commands.